hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/throttle/StoreHotnessProtector.java

```diff
@@ -118,7 +119,12 @@ private static void logDisabledMessageOnce() {
   }
 
   public void update(Configuration conf) {
-    init(conf);
+    // Add table descriptor configuration
+    CompoundConfiguration newconf = new CompoundConfiguration().add(conf);
+    if (this.region != null) {
+      newconf.addBytesMap(this.region.getTableDescriptor().getValues());
+    }
+    init(newconf);
```

Comment:
Will we call the init method from other places? Do we also use a CompoundConfiguration there?